--- libdrm-2.3.0/libdrm.pc.in	2008-10-31 06:38:56.000000000 -0700
+++ libdrm-2.3.0/libdrm.pc.in.new	2009-05-13 15:51:25.092858114 -0700
@@ -7,4 +7,4 @@
 Description: Userspace interface to kernel DRM services
 Version: @PACKAGE_VERSION@
 Libs: -L${libdir} -ldrm
-Cflags: -I${includedir} -I${includedir}/drm
+Cflags: -I${includedir}/psb -I${includedir}/psb/drm
--- libdrm-2.3.0/shared-core/Makefile.am	2009-05-13 13:45:38.563871125 -0700
+++ libdrm-2.3.0/shared-core/Makefile.am.new	2009-05-13 15:50:57.903870870 -0700
@@ -22,7 +22,7 @@
 # however, r300 and via need their reg headers installed in order to build.
 # better solutions are welcome.
 
-klibdrmincludedir = ${includedir}/drm
+klibdrmincludedir = ${includedir}/psb/drm
 klibdrminclude_HEADERS = \
                          drm.h \
                          drm_sarea.h \
--- libdrm-2.3.0/libdrm/Makefile.am	2008-11-18 01:01:45.000000000 -0800
+++ libdrm-2.3.0/libdrm/Makefile.am.new	2009-05-13 15:51:02.500871356 -0700
@@ -25,7 +25,7 @@
 AM_CFLAGS = -I$(top_srcdir)/shared-core
 libdrm_la_SOURCES = xf86drm.c xf86drmHash.c xf86drmRandom.c xf86drmSL.c xf86drmMode.c
 
-libdrmincludedir = ${includedir}
+libdrmincludedir = ${includedir}/psb
 libdrminclude_HEADERS = xf86drm.h xf86mm.h xf86drmMode.h
 
 EXTRA_DIST = ChangeLog TODO
